extensions/amp-auto-lightbox/0.1/amp-auto-lightbox.js

```diff
@@ -273,10 +273,11 @@ function candidateSelector(tagName) {
 
 /**
  * @param {!Element} element
- * @return {!Promise<!Element>}
+ * @return {!Promise}
  */
 function whenLoaded(element) {
-  return element.signals().whenSignal(CommonSignals.LOAD_END);
```

Comment:
Hmm, this is a bug with our externs. `signals` isn't on `Element`, only an `AmpElement`. We'll likely catch other bugs.